library/src/scala/quoted/Quotes.scala

```diff
@@ -2049,6 +2054,56 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     // ----- Patterns ------------------------------------------------
 
+    /** Pattern representing a `_` wildcard. */
+    type Wildcard <: Tree
```

Comment:
If it's only used in patterns maybe call it PatternWildcard to avoid confusing it with other wildcards.

Reply:
Updated to `WildcardPattern`